test/utils/tree-view/describeTreeView/describeTreeView.tsx

```diff
@@ -14,15 +14,15 @@ import {
   DescribeTreeViewItem,
 } from './describeTreeView.types';
 
-const innerDescribeTreeView = <TPlugin extends TreeViewAnyPluginSignature>(
+const innerDescribeTreeView = <TPlugins extends TreeViewAnyPluginSignature[]>(
```

Comment:
I now pass an array of plugins. I needed it to access `defaultExpandedItems` in `useTreeIcons.test.tsx`